<a href="https://colab.research.google.com/github/GijooHong/BP_05_Ch3/blob/main/basic_python221028.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

1.사용자로부터 두 개의 정수를 받은 정수의 합, 정수의 차, 정수의 곱, 정수의 평균, 큰 수, 작은 수를 계산하여 화면에 출력하는 프로그랩을 작성하라. 파이썬이 제공하는 내 장 함수 max(x, y), min(x, y)을 사용해보자.

*   항목 추가
*   항목 추가



In [ ]:
x = int(input("x의 값은?: "))
y = int(input("y의 값은?: "))

print("두 수의 합은 :", x + y)
print("두 수의 차는 :", x - y)
print("두 수의 곱은 :", x * y)
print("두 수의 평균은 :", (x + y)/2)
print("큰 수는? :", max(x, y))
print("작은 수는? :", min(x, y))

x의 값은?: 9
y의 값은?: 6
두 수의 합은 : 15
두 수의 차는 : 3
두 수의 곱은 : 54
두 수의 평균은 : 7.5
큰 수는? : 9
작은 수는? : 6


2.원기둥의 부피를 계산하는 프로그램 을 작성해보자. 원기둥의 부피는 다음과 같이 계산 한다.(계산식은 3.141592 r**2 h)

In [ ]:
r =float(input("r값은? :"))
h =float(input("h값은? :"))

vol = 3.141592 * r**2 * h
print("원기둥의 부피:", vol)

r값은? :10
h값은? :10
원기둥의 부피: 3141.592


3. 사용자로부터 정수를 받아서 정수의 자리수의 합을 계산하는 프로그램을 작성하여 보자. 예를 들어서 사용자가 1234를 입력하였다면 1+2+3+4를 계산하면 된다. 나머지 연산자와 정수 나눗셈 연산자를 //를 적극적으로 사용해보자.

In [ ]:
number = int(input("정수를 입력하시오.: "))

sum = 0
sum = sum + number % 10
number = number // 10

sum = sum + number % 10
number = number // 10

sum = sum + number % 10
number = number // 10

sum = sum + number % 10
number = number // 10

print("자리수의 합은: ", str(sum) )

정수를 입력하시오.: 1234
자리수의 합은:  10


4. 사용자로부터 두 점의 좌표 (x1,y1)과 (x2,y2)를 입력받아서 두 점 사이의 거리를 계산하는 프로그램을 작성해보자. 스크립트 모드로 작성하라. 거리는 다음 식으로 계산한다. 

In [ ]:
x1 = int(input("x1값을 입력하시오.: "))
x2 = int(input("x2값을 입력하시오.: "))
y1 = int(input("y1값을 입력하시오.: "))
y2 = int(input("y2값을 입력하시오.: "))
dist = ((x2 - x1)**2 + (y2 - y1)**2)** 0.5
print("두점의 사이의 거리는? ", dist, "입니다.")

x1값을 입력하시오.: 0
x2값을 입력하시오.: 100
y1값을 입력하시오.: 0
y2값을 입력하시오.: 100
두점의 사이의 거리는?  141.4213562373095 입니다.


5. 

import turtle 
t = turtle.Turtle()
t.shape("turtle")
t.left(45)
t.forward(141)
t.setheading(0)
t.goto(0,0)
t.fd(100)
t.lt(90)
t.fd(100)

6.

In [ ]:
import turtle
t= turtle.Turtle()
t.shape("turtle")
x1 = int(input("x1:"))
x1:0
y1 = int(input("y1:"))
y1:0
x2 = int(input("x2:"))
x2:100
y2 = int(input("y2:"))
y2:100
dist = ((x1 - x2)**2 +(y1 - y2)**2)**0.5
t.up()
t.goto(x1,y1)
t.down()
t.goto(x2,y2)
t.write("점의 길이는=" + str(dist))

7.

In [6]:
import time
fseconds = time.time()
sec = int(fseconds)
min = (sec//60)%60
hour = ((sec//60)//60)%24

print("현재 시간(영국 그리니치 시각):" +str(hour+9) + "시" + str(min) + "분")

현재 시간(영국 그리니치 시각):19시48분


8.

In [7]:
weight=int(input("물체의 무게를 입력하시오(킬로그램) :"))
speed=int(input("물체의 속도를 입력하시오(미터/초) :"))
print("물체는", 0.5*weight*speed**2, "(줄)의 에너지를 가지고 있다.")

물체의 무게를 입력하시오(킬로그램) :10
물체의 속도를 입력하시오(미터/초) :100
물체는 50000.0 (줄)의 에너지를 가지고 있다.
